config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
 Style/Documentation:
   Enabled: false
 
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
 Style/EmptyLinesAroundClassBody:
   Enabled: false
 
 Style/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Style/ExtraSpacing:
+  Enabled: false
+
+Style/IndentArray:
```

Comment:
You say that other styles here work fine for you, but are you also saying that you have a strong objection to leaving this enabled? If you don't have a strong objection, I'd say let's leave it alone.

(I ask because this rule should be easily enforceable via our editors, and autofix should be able to help out where our editors fail us.)